codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
```

Comment:
Once again, import what you need.

Also, I'm curious: do all OS X environments have access to this `Cocoa` module? My machine doesn't have it:

```
$ python
Python 2.7.10 (default, Jul  3 2015, 01:23:30)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import Cocoa
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named Cocoa
```